app/views/layouts/application.html.haml

```diff
@@ -26,7 +26,7 @@
         a.async = 1;
         a.src = g;
         m.parentNode.insertBefore(a, m)
-      })(window, document, 'script', 'https://storage.googleapis.com/cdn-alg-insights/beta/v1/alg-insights.js', 'aa')
+      })(window, document, 'script', 'https://cdn.jsdelivr.net/npm/alg-insights/dist/alg-insights.js', 'aa')
```

Comment:
@JonasBa could we tag it `0.x.y` in order to make sure it doesn't get auto-updated?

Reply:
Yes, indeed. Please use `https://cdn.jsdelivr.net/npm/alg-insights@0.0.2/dist/alg-insights.js`

Reply:
@JonasBa this version targets `"http://localhost:8080/1/"` :/

Reply:
Yeah, bugged when deploying. You can use 0.0.3 and it's fixed